addons/base_import/static/src/import_action/import_action.js

```diff
@@ -185,7 +185,7 @@ export class ImportAction extends Component {
                 sprintf(this.env._t("%s records successfully imported"), res.ids.length),
                 { type: "success" }
             );
-            this.exit();
+            this.exit(res.ids);
```

Comment:
Would be cleaner to specify the argument in the signature of the method #justJSthings